src/cpp/ext/filters/census/server_call_tracer.cc

```diff
@@ -160,6 +160,13 @@ class OpenCensusServerCallTracer : public grpc_core::ServerCallTracer {
     context_.AddSpanAnnotation(annotation, {});
   }
 
+  void RecordAnnotation(const Annotation& annotation) override {
+    switch (annotation.type()) {
+      default:
+        context_.AddSpanAnnotation(annotation.ToString(), {});
```

Comment:
Same cost analysis... maybe move the check in here?

Reply:
Updated.